ui/app/src/models/MarketplacePlugin.ts

```diff
@@ -31,17 +31,12 @@ interface Media {
   videos?: [Video] | [];
 }
 
-interface Version {
+export interface Version {
```

Comment:
Rename **all** models here to something like `MarketplacePluginVersion`. We need to remove these vague names that can correspond to a bunch of different concepts of the system.